dangerfile.js

```diff
@@ -28,12 +28,22 @@ import {
   propKeys as iosPropKeys,
 } from './packages/bpk-tokens/tokens/base.raw.ios.json';
 
+import * as meta from './meta.json';
+
+const BACKPACK_SQUAD_MEMBERS = meta.maintainers.map(
+  maintainer => maintainer.github,
+);
+const author = danger.github.pr.user.login;
+const isPrExternal = !BACKPACK_SQUAD_MEMBERS.includes(author);
```

Comment:
This could probably be terser by doing it all in one loop, but I feel it's more readable this way, and more open to any future uses.